inscriptions/ISic3092.xml

```diff
@@ -177,25 +182,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331418">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — —</supplied>ΤΩΝ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">— — —</supplied>ΗΘΗΗ
-                    <supplied reason="lost">— τε͂ς μηνὸς</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">Ἰουλί</supplied>ου δέ
-                    <supplied reason="lost">κα ἐν τῇ</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">ὑπατί</supplied>ᾳ Βα
-                    <supplied reason="lost">λεντ. καὶ Βαλεντ. —ʹ</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΤΩΝ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΗΘΗΗ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><supplied reason="lost">τε͂ς</supplied> <supplied reason="lost">μηνὸς</supplied> <supplied reason="lost">ἰουλί</supplied>ου δέ<supplied reason="lost">κα</supplied> <supplied reason="lost">ἐν</supplied> <supplied reason="lost">τῇ</supplied>
```

Comment:
Tag the number even though it's within a date expression? We decided to put off actual tagging of whole date expressions for now, though, I think?